recipe/meta.yaml

```diff
@@ -59,6 +58,7 @@ requirements:
     - ocl-icd  # [linux64]
     - clhpp  # [linux64 or win64]
     - khronos-opencl-icd-loader  # [win64]
+    - yaml-cpp  # [linux64]
```

Comment:
NPU requires subdirectory https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_npu/thirdparty/CMakeLists.txt#L21

We need to add `find_package` support to NPU thirdparty